tools/analyzer_plugin/test/unit/fluent_interface_util/cascade_read_test.dart

```diff
@@ -0,0 +1,81 @@
+import 'package:over_react_analyzer_plugin/src/fluent_interface_util.dart';
+import 'package:test/test.dart';
+
+import '../../test_util.dart';
+
+main() {
+  group('ast_util', () {
+    group('FluentComponentUsage.cascadedProps', () {
+      test('returns cascaded props', () {
+        final usage = parseAndGetComponentUsage(/*language=dart*/ r'''
```

Comment:
These are [language injection comments](https://www.jetbrains.com/help/webstorm/using-language-injections.html#use-language-injection-comments) for JetBrains IDEs, which give you basic syntax highlighting/checking for the strings.

This is especially useful to alert you of whether the code you're passing in has syntax errors without having to run the tests to find out.